### PR TITLE
add custom class to hide duplicated docs notification

### PR DESCRIPTION
**Pull request by ETLaurent**
## Summary

Hide the "Duplicates detected" warning.

Relies on https://github.com/apostrophecms/apostrophe/pull/4334

### CODE REVIEW

**Comment by ETLaurent** [comment]
tests running https://github.com/apostrophecms/testbed/pull/208

File: lib/methods/import.js [hunk 1]
```diff
@@ -82,7 +82,8 @@ module.exports = self => {
         event: {
           name: 'import-duplicates',
           data: results
-        }
+        },
+        classes: [ 'apos-notification--hidden' ]
```
Comment (ValJed) [inline lib/methods/import.js:86]: I think you could deleted the translation key that were used here, and simply pass an empty string as second parameter?

**Review by ValJed — CHANGES_REQUESTED** [review]
Minor change otherwise straightforward 👍🏼

File: lib/methods/import.js (hunk 1 above)
Comment (ETLaurent) [reply lib/methods/import.js:86]: translation key is also used in the modal

**Review by ValJed — APPROVED** [review]

**Pull request merged**